src/Nethermind/Nethermind.Consensus/Processing/ProcessingStats.cs

```diff
@@ -92,7 +93,8 @@ public void UpdateStats(Block? block, IBlockTree blockTreeCtx, int recoveryQueue
             long runningMicroseconds = _runStopwatch.ElapsedMicroseconds();
             decimal runMicroseconds = (runningMicroseconds - _lastElapsedRunningMicroseconds);
 
-            if (runMicroseconds > 1 * 1000 * 1000)
+            long reportMs = Environment.TickCount64;
+            if (reportMs - _lastReportMs > 1000)
```

Comment:
Why not https://learn.microsoft.com/en-us/dotnet/api/system.diagnostics.stopwatch.getelapsedtime?view=net-7.0#system-diagnostics-stopwatch-getelapsedtime(system-int64-system-int64) ?

Reply:
Why? `Environment.TickCount64` will take 292 277 266 years to wrap and is already in milliseconds; maths is just 1000 for a second and is only checking if a second has passed since last time, no need for sub nanosecond precision?

https://learn.microsoft.com/en-us/dotnet/api/system.environment.tickcount64?view=net-7.0